pshell/tests/test_procs.py

```diff
@@ -95,8 +99,57 @@ def test_kill2():
         sh.kill('foo')
 
 
-@pytest.mark.skip('TODO')
-def test_sigkill():
-    """Test terminating processes resilient to SIGTERM
+@pytest.mark.skipif(
+    os.name == 'nt',
+    reason='On Windows, os.kill() and psutil.kill() calls TerminateProcess '
+           'API which does not process signals (such as SIGTERM, SIGKILL '
+           'etc..) as ANSI/POSIX prescribed.  The TerminateProcess API '
+           'unconditionally terminates the target process.')
+def test_sigkill_sigterm_delay5():
+    """Test that kill() will send a SIGTERM to kill the target first.  Process
+    that shuts itself downupon receiving SIGTERM will be able to do so
+    gracefully.
     """
-    pass
+    cmd = ['python', os.path.join(DATADIR, 'sleep20_sigterm_delay5.py')]
```

Comment:
This is problematic when the desired interpreter is not in the PATH. This is actually the norm when running the test from inside PyCharm. Please replace 'python' with sys.executable.